src/models/steps/data_validation/common/object_detection_dataset_context_validator.py

```diff
@@ -62,16 +62,30 @@ def _validate_bounding_box_coordinates(self, coco_file_annotation: Annotation):
             for coco_file_image in self.dataset_context.coco_file.images
             if coco_file_image.id == coco_file_annotation.image_id
         ][0]
-        return all(
-            [
-                x >= 0,
-                y >= 0,
-                width > 0,
-                height > 0,
-                x + width <= coco_file_image.width,
-                y + height <= coco_file_image.height,
-            ]
-        )
+        if x < 0:
```

Comment:
Instead:
- Count each error type + keep track of the annotations to fix
- Print the Warnings (⚠️)
- Add an extra parameter to the experiment, e.g. `fix_annotation`
- If true: fix, else raise ValueError("...")